dist/ng-formio-helper.js

```diff
@@ -670,25 +782,28 @@ angular.module('ngFormioHelper', ['formio', 'ngFormioGrid', 'ui.router'])
         setAppUrl: function(url) {
           FormioProvider.setAppUrl(url);
         },
-        register: function (name, resource, path) {
+        register: function (name, resource, path, form, override) {
+          var shouldOverride = form && !override;
           if (!registered) {
             registered = true;
             $stateProvider.state('auth', {
               abstract: true,
               url: '/auth',
-              templateUrl: 'views/user/auth.html'
+              templateUrl: shouldOverride ? 'formio-helper/auth/auth.html' : 'views/user/auth.html'
             });
           }
 
           if (!path) {
             path = name;
           }
+          var tpl = name.toLowerCase() + '.html';
           $stateProvider
             .state('auth.' + name, {
               url: '/' + path,
               parent: 'auth',
-              templateUrl: 'views/user/' + name.toLowerCase() + '.html',
+              templateUrl: shouldOverride ? 'formio-helper/auth/' + tpl : 'views/user/' + tpl,
               controller: ['$scope', '$state', '$rootScope', function ($scope, $state, $rootScope) {
+                $scope.currentForm = form;
```

Comment:
I don't think this line was necessary. I was using it to show the last modified date of the currently displayed form. Not needed for offline mode.